### PR TITLE
Core packages in OSS

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | n/a
| Versions      | 3.3+

Based on feedback from the community, adds a note indicating which packages are available in the OSS edition.

### CODE REVIEW

File: docs/guide/bundles.md [hunk 1]
```diff
@@ -29,6 +29,10 @@ see the [How to Remove a Bundle](http://symfony.com/doc/5.0/bundles/remove.html)
 
 ## Core packages
 
+!!! tip
+
+    The core packages are included in Ibexa Open Source.
```
Comment (dabrt) [inline docs/guide/bundles.md:34]: Howabout using the active voice?
"Ibexa Open Source includes the core packages."?

**Review by dabrt — APPROVED** [review]

File: docs/guide/bundles.md (hunk 1 above)
Comment (DominikaK) [reply docs/guide/bundles.md:34]: I've tried to make it clear that core packages are the *only* ones in OSS, without actually saying "only" :)

Comment (dabrt) [reply docs/guide/bundles.md:34]: In this case, I like "Ibexa Open Source is composed of the core packages" best, despite its passivevoiciness

Comment (DominikaK) [inline docs/guide/bundles.md:34]:
```suggestion
    Ibexa Open Source is composed of the core packages.
```

**Pull request merged**